beacon_node/network/src/sync/range_sync/chain.rs

```diff
@@ -1013,7 +1015,8 @@ impl<T: BeaconChainTypes> SyncingChain<T> {
                     chain_id: self.id,
                     batch_id,
                 },
-                &synced_peers,
+                &self.peers,
+                &synced_column_peers,
```

Comment:
```suggestion
                // Request blocks only from peers of this specific chain
                &self.peers,
                // Request column from all synced peers, even if they are not part of this chain. Since we request by root
                // this is safe. We request from peers outside the chain to reduce the chance of getting stuck in the case
                // this chain is useful but it has very few peers that agree on the exact same head root.
                &synced_column_peers,
```